### ISSUE TITLE
Manually empty the trashbin

### ISSUE THREAD

**Issue by DemoFreak** [nextcloud/deck#4274]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

**Describe the solution you'd like**
It would be useful to have a way to manually remove the deleted objects (i.e. empty the trashbin). Until now, they have been in the "Deleted objects" tab for all eternity and are not cleaned up even by the cronjob.

**Comment by putt1ck** [comment]
Like
`sudo -u www-data php /var/www/nextcloud/occ trashbin:cleanup`

**Comment by DemoFreak** [comment]
Yes, but for Deck and not for Files. And with a pretty button in the GUI.

**Comment by user8446** [comment]
I was going to open a new issue but found this.

Deleted cards will purge as expected from the deleted items list, but deleted lists do remain forever - whether the list has cards in it or is empty. Even though this was opened 11/23/22 this behavior still persists today with version 1.11.4
